src/js/overlay.js

```diff
@@ -9,11 +9,6 @@ const checkOptions = function(opts) {
 		opts.trigger = document.querySelector(opts.trigger);
 	}
 
-	// There can't be a heading with an empty title
```

Comment:
Should we still throw this error if `visuallyHideTitle` is set to `false` and `heading.title` is an empty string?